pkg/api/vlabs/validate_test.go

```diff
@@ -1647,12 +1647,73 @@ func TestProperties_ValidateZones(t *testing.T) {
 	tests := []struct {
 		name                string
 		orchestratorVersion string
+		masterProfile       *MasterProfile
 		agentProfiles       []*AgentPoolProfile
 		expectedErr         string
 	}{
+		{
+			name:                "Master profile with zones version",
+			orchestratorVersion: "1.11.3",
```

Comment:
we should use `orchestratorRelease` in unit tests whenever possible as individual patches will get deprecated as we add new ones (we now support the two latest patches of each minor version)